### ISSUE TITLE
office-addin-debugging expects questionable path for webextension.xml

### ISSUE THREAD

**Issue by dexterouschen** [OfficeDev/Office-Addin-Scripts#499]
## Expected Behavior

`office-addin-debugging start manifest.xml --document "/path/to/document"` should work as expected 

## Current Behavior

`office-addin-debugging start manifest.xml --document "/path/to/document"` should work as documented but since

https://github.com/OfficeDev/Office-Addin-Scripts/blob/c02a4c95fc868a65ddce15d1a278fc0fb18a76d6/packages/office-addin-dev-settings/src/sideload.ts#L186

indicates that all Excel add-ins must use `xl/webextensions/webextension.xml`, any Excel document without `xl/webextensions/webextension.xml` will simply fail at

https://github.com/OfficeDev/Office-Addin-Scripts/blob/c02a4c95fc868a65ddce15d1a278fc0fb18a76d6/packages/office-addin-dev-settings/src/sideload.ts#L78

I'm working on a `TaskPane` app so according to

https://github.com/OfficeDev/Office-Addin-Scripts/blob/c02a4c95fc868a65ddce15d1a278fc0fb18a76d6/packages/office-addin-dev-settings/src/sideload.ts#L158

the default template for that is `https://github.com/OfficeDev/Office-Addin-Scripts/blob/master/packages/office-addin-dev-settings/templates/ExcelWorkbookWithTaskPane.xlsx`

`ExcelWorkbookWithTaskPane.xlsx` does have `xl/webextensions/webextension.xml` within the file content.

However, all of my newly created `xlsx` files have `xl/webextensions/webextension1.xml` instead of `xl/webextensions/webextension.xml` within the file content.

If I run `office-addin-debugging start manifest.xml` to open the renamed default `ExcelWorkbookWithTaskPane.xlsx` template, and save as a new `xlsx`, it contains `xl/webextensions/webextension1.xml` instead as well.

I'm not sure if this is an Excel version issue, local config issue, or something else.

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Operating System: macOS Big Sur 11.5.1
* Node version: 14.17.4
* Office version: Microsoft Excel for Mac 16.51
* Tool version: office-addin-debugging 4.2.2

**Comment by akrantz** [comment]
The ability to sideload to Office desktop requires the document to have a webextension part. If you sideload without specifying the --document parameter, the document created has such a webextension part. You can then reuse that document and refer to it via the --document parameter when sideload to the web.

I've created a bug to clarify and track that it is currently not supported to sideload to desktop for a document without the webextension part. https://github.com/OfficeDev/Office-Addin-Scripts/issues/500

**Comment by akrantz** [comment]
It would be good to understand if you have an Excel document that has different or multiple webextension parts. If you have steps to reproduce it or can provide an example document without any personal information, it might help to understand your scenario better. Since you closed the bug it sounds like you found another way to accomplish what you're trying to do. We intentionally didn't do the work to support arbitrary documents when we added the --document parameter, which was mainly intended for sideload to web, but wanted to allow for the ability to implement this based on user feedback such as yours if we found cases where there was sufficient need to do so.
